code/game/machinery/computer/crew.dm

```diff
@@ -199,7 +199,14 @@ var/global/datum/crewmonitor/crewmonitor = new
 	for (z in src.interfaces)
 		if (src.interfaces[z] == hi) break
 
-	return (hclient.client.mob && hclient.client.mob.stat == 0 && hclient.client.mob.z == text2num(z) && ( isAI(hclient.client.mob) || (locate(/obj/machinery/computer/crew, range(1, hclient.client.mob))) || (locate(/obj/item/device/sensor_device, hclient.client.mob.contents)) ) )
+	if (hclient.client.mob && hclient.client.mob.stat == 0 && hclient.client.mob.z == text2num(z))
+		if (isAI(hclient.client.mob)) return TRUE
+		else if (isrobot(hclient.client.mob)) // fix for /tg/station issue #10380 to allow borgs to operate the crew monitoring computer remotely

```

Comment:
don't do this.
We don't comment on the bugs we fix with the actual issue report.
Not to mention it's clearly readable from this code that it's designed to make robots able to use it at range.

tl;dr remove the comment.